### DISCUSSION TITLE
Defaults for maxConcurrentActivityFunctions and maxConcurrentOrchestratorFunctions on isolated plan

### DISCUSSION THREAD

**Discussion by czeslav87** [Azure/azure-functions-durable-extension#2560] [Q&A] [answered]
Documentation states:

maxConcurrentActivityFunctions | Consumption plan: 10Dedicated/Premium plan: 10X the number of processors on the current machine | The maximum number of activity functions that can be processed concurrently on a single host instance.
-- | -- | --
maxConcurrentOrchestratorFunctions | Consumption plan: 5Dedicated/Premium plan: 10X the number of processors on the current machine | The maximum number of orchestrator functions that can be processed concurrently on a single host instance.

But what are the default values for Isolated Plan?

**Comment by cgillum** [comment, marked as answer]
The isolated plan is a type of dedicated plan, so you can refer to the dedicated numbers.

Generally speaking, there are just two categories of apps we consider: Consumption and non-Consumption. The language used for "non-Consumption" in this case was "Dedicated/Premium", which can be a bit confusing.

**Reply by czeslav87** [reply]
Thank you Chris, that makes sens.
